### PR TITLE
Prettify repository using prettier for formatting consistency throughout.

### PR DESCRIPTION
**Pull request by accountingnerd**
This PR introduces Prettier (https://prettier.io/) for consistent code formatting within the repository.  I think the docs do a pretty good job of explaining why: https://prettier.io/docs/en/why-prettier.

If this is not something that the team is keen to do, then feel free to ignore/drop this request. No hurt feelings on my part. Just something that I've been doing in all my projects for the past few years.

I, personally, can't (or rather, don't want to) go back to life without prettier. :)

### CODE REVIEW

**Comment by MrRob** [comment]
@accountingnerd I do appreciate consistent formatting.  Let me explore this a bit.  In the meantime, there is now a conflict here from another PR, can you resolve that?

File: example-ClientScript.ts [hunk 1]
```diff
@@ -3,32 +3,44 @@
...
 
-  const customerId = ctx.currentRecord.getValue('entity'); // Assume this script is running on a transaction
-  search.lookupFields.promise({ type: 'customer', id: customerId, columns: ['companyname', 'datecreated', 'entitystatus'] }).then((values) => {
-    const name   = values.companyname as string;
-    const date   = values.datecreated as string;
-    const status = values.entitystatus as { value: string, text: string }[];
-    console.log('Customer', name, 'created at', date, 'status', status);
-  });
+    const customerId = ctx.currentRecord.getValue('entity'); // Assume this script is running on a transaction
+    search.lookupFields
```
Comment (MrRob) [inline example-ClientScript.ts:13]: @accountingnerd Do you know if there is a way to set it not to break out the line like this?  We prefer the way it was before, on a single line

Comment (accountingnerd) [reply example-ClientScript.ts:13]: Sure. If you include a comment `// prettier-ignore` in the file, prettier will ignore the next node in the AST.

I'm pushing up a commit right now with that on this file so you can see.

**Comment by accountingnerd** [comment]
Hey @MrRob - any further thoughts on this? If I'm pestering you too much, just say so. I'm not keen to be a pest. I just see a reminder that this is still open in VS Code every morning. :)

**Comment by MrRob** [comment]
I'm not really a fan of how much of the styling it automates.  Maybe there is a middle-ground here, I haven't had a chance to really dig into it to find out

**Comment by accountingnerd** [comment]
Sounds like this may not be a good fit for you, then. It's a very opinionated formatter, and there aren't a ton of settings. https://prettier.io/docs/en/option-philosophy.

If/when you're ready to take the plunge, let me know. I'd be happy to help you.

**Pull request closed without merging**